apps-rendering/src/components/Layout/LiveLayout.tsx

```diff
@@ -186,6 +186,8 @@ const LiveLayout: FC<Props> = ({ item }) => {
 						<LiveBlocks
 							blocks={item.pagedBlocks.currentPage.blocks}
 							format={item}
+							pinnedPost={item.pinnedBlock}
```

Comment:
Good idea @georgeblahblah 

Because we are using the shared component `LiveBlockContainer` and in there the relevant properties are called `isPinnedPost` & `isOriginalPinnedPost` I changed the field in the `item` model to be `pinnedPost` rather than `pinnedBlock` to make the naming consistent. Do you think it's OK?